src/styles/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  padding: 20px 10px;
+  width: 7 * ($sell-size + $sell-gap);
+  border: 2px solid black;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @each $day, $margin in $days {
+    &--start-day--#{$day} {
+      .calendar__sell:first-child {
+        margin-left: $margin;
+      }
+    }
+  }
+
+  @for $month-length from 28 through 31 {
+    &--month-length--#{$month-length} {
+      .calendar__sell:nth-child(n + #{$month-length + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__sell {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $sell-size;
+    height: $sell-size;
+    border: 1px solid black;
+    position: relative;
+    background-color: $Sell-color;
+    transition: all 0.5s;
```

Comment:
list styles that you apply transition to instead of "all"

Reply:
okay